Release fails on integration tests

Release 0.2.0 failed because it could not find the `integration-tests` pom artifact during the `release:perform` step.
In the main pom we have:
```
    <profiles>
        <profile>
            <id>it</id>
            <activation>
                <property>
                    <name>performRelease</name>
                    <value>!true</value>
                </property>
            </activation>
            <modules>
                <module>integration-tests</module>
            </modules>
        </profile>
    </profiles>
```
I do not remember why we had that? is this related to native tests?

and the command is:
```
mvn -B release:perform -Darguments=-DperformRelease -Prelease -s maven-settings.xml
```

so you would think that the tests are skipped during `release:perform`
yet the log shows:
```
2021-02-17T20:50:15.5078850Z [INFO] ------------------------------------------------------------------------
2021-02-17T20:50:15.5080521Z [INFO] Reactor Build Order:
2021-02-17T20:50:15.5081714Z [INFO] 
2021-02-17T20:50:15.5090494Z [INFO] Quarkus - Freemarker - Parent                                      [pom]
2021-02-17T20:50:15.5094972Z [INFO] Quarkus - Freemarker - Runtime                                     [jar]
2021-02-17T20:50:15.5099658Z [INFO] Quarkus - Freemarker - Deployment                                  [jar]
2021-02-17T20:50:15.5104052Z [INFO] Quarkus - Freemarker - Integration tests                           [pom]
2021-02-17T20:50:15.5108500Z [INFO] Quarkus - Freemarker - Integration test - Configuration defaults   [jar]
2021-02-17T20:50:15.5113173Z [INFO] Quarkus - Freemarker - Integration test - Legacy configuration     [jar]
2021-02-17T20:50:15.5117893Z [INFO] Quarkus - Freemarker - Integration test - Template sets            [jar]
```

and the build fails with:
```
2021-02-17T20:53:23.1212055Z [INFO] [INFO] -------------------------------------------------------
2021-02-17T20:53:23.1213577Z [INFO] [INFO]  T E S T S
2021-02-17T20:53:23.1215002Z [INFO] [INFO] -------------------------------------------------------
2021-02-17T20:53:23.9827140Z [INFO] [INFO] Running io.quarkiverse.it.freemarker.ConfigDefaultsTest
2021-02-17T20:53:24.6593251Z [INFO] Downloading from central: https://repo.maven.apache.org/maven2/io/quarkiverse/freemarker/quarkiverse-freemarker-integration-tests/0.2.0/quarkiverse-freemarker-integration-tests-0.2.0.pom
2021-02-17T20:53:25.0650447Z [INFO] [ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 1.044 s <<< FAILURE! - in io.quarkiverse.it.freemarker.ConfigDefaultsTest
2021-02-17T20:53:25.0652951Z [INFO] [ERROR] dynamicTemplate  Time elapsed: 0.017 s  <<< ERROR!
2021-02-17T20:53:25.0657235Z [INFO] java.lang.RuntimeException: io.quarkus.bootstrap.BootstrapException: Failed to create the application model for io.quarkiverse.freemarker:quarkiverse-freemarker-integration-test-config-defaults::jar:0.2.0
2021-02-17T20:53:25.0661715Z [INFO] 	at io.quarkus.test.junit.QuarkusTestExtension.throwBootFailureException(QuarkusTestExtension.java:655)
2021-02-17T20:53:25.0665779Z [INFO] 	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestClassConstructor(QuarkusTestExtension.java:728)
2021-02-17T20:53:25.0669564Z [INFO] 	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
2021-02-17T20:53:25.0673421Z [INFO] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
2021-02-17T20:53:25.0678052Z [INFO] 	at org.junit.jupiter.api.extension.InvocationInterceptor.interceptTestClassConstructor(InvocationInterceptor.java:72)
2021-02-17T20:53:25.0682891Z [INFO] 	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
2021-02-17T20:53:25.0686889Z [INFO] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
2021-02-17T20:53:25.0692151Z [INFO] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
2021-02-17T20:53:25.0696550Z [INFO] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
2021-02-17T20:53:25.0700921Z [INFO] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
2021-02-17T20:53:25.0704623Z [INFO] 	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
2021-02-17T20:53:25.0707953Z [INFO] 	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:77)
2021-02-17T20:53:25.0712322Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeTestClassConstructor(ClassBasedTestDescriptor.java:342)
2021-02-17T20:53:25.0717466Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateTestClass(ClassBasedTestDescriptor.java:289)
2021-02-17T20:53:25.0722090Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.instantiateTestClass(ClassTestDescriptor.java:79)
2021-02-17T20:53:25.0746828Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateAndPostProcessTestInstance(ClassBasedTestDescriptor.java:267)
2021-02-17T20:53:25.0752260Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$2(ClassBasedTestDescriptor.java:259)
2021-02-17T20:53:25.0755382Z [INFO] 	at java.base/java.util.Optional.orElseGet(Optional.java:369)
2021-02-17T20:53:25.0758590Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$3(ClassBasedTestDescriptor.java:258)
2021-02-17T20:53:25.0763187Z [INFO] 	at org.junit.jupiter.engine.execution.TestInstancesProvider.getTestInstances(TestInstancesProvider.java:31)
2021-02-17T20:53:25.0868321Z [INFO] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$prepare$0(TestMethodTestDescriptor.java:101)
2021-02-17T20:53:25.0872597Z [INFO] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2021-02-17T20:53:25.0876718Z [INFO] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:100)
2021-02-17T20:53:25.1007991Z [INFO] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:65)
2021-02-17T20:53:25.1012000Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$prepare$1(NodeTestTask.java:111)
2021-02-17T20:53:25.1016138Z [INFO] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2021-02-17T20:53:25.1020148Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.prepare(NodeTestTask.java:111)
2021-02-17T20:53:25.1023815Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:79)
2021-02-17T20:53:25.1026519Z [INFO] 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
2021-02-17T20:53:25.1031010Z [INFO] 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2021-02-17T20:53:25.1036354Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
2021-02-17T20:53:25.1040339Z [INFO] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2021-02-17T20:53:25.1044341Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
2021-02-17T20:53:25.1128251Z [INFO] 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
2021-02-17T20:53:25.1131738Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
2021-02-17T20:53:25.1135723Z [INFO] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2021-02-17T20:53:25.1139934Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
2021-02-17T20:53:25.1143888Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
2021-02-17T20:53:25.1146610Z [INFO] 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
2021-02-17T20:53:25.1151134Z [INFO] 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2021-02-17T20:53:25.1156487Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
2021-02-17T20:53:25.1160501Z [INFO] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2021-02-17T20:53:25.1164490Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
2021-02-17T20:53:25.1248258Z [INFO] 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
2021-02-17T20:53:25.1251757Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
2021-02-17T20:53:25.1255762Z [INFO] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2021-02-17T20:53:25.1260353Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
2021-02-17T20:53:25.1264329Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
2021-02-17T20:53:25.1269804Z [INFO] 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
2021-02-17T20:53:25.1275464Z [INFO] 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
2021-02-17T20:53:25.1280041Z [INFO] 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
2021-02-17T20:53:25.1283856Z [INFO] 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
2021-02-17T20:53:25.1287111Z [INFO] 	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
2021-02-17T20:53:25.1290709Z [INFO] 	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
2021-02-17T20:53:25.1323722Z [INFO] 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
2021-02-17T20:53:25.1327088Z [INFO] 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
2021-02-17T20:53:25.1361987Z [INFO] 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
2021-02-17T20:53:25.1376311Z [INFO] 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
2021-02-17T20:53:25.1380541Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
2021-02-17T20:53:25.1384443Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
2021-02-17T20:53:25.1387618Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
2021-02-17T20:53:25.1390448Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2021-02-17T20:53:25.1395466Z [INFO] Caused by: io.quarkus.bootstrap.BootstrapException: Failed to create the application model for io.quarkiverse.freemarker:quarkiverse-freemarker-integration-test-config-defaults::jar:0.2.0
2021-02-17T20:53:25.1399733Z [INFO] 	at io.quarkus.bootstrap.BootstrapAppModelFactory.resolveAppModel(BootstrapAppModelFactory.java:320)
2021-02-17T20:53:25.1403068Z [INFO] 	at io.quarkus.bootstrap.app.QuarkusBootstrap.bootstrap(QuarkusBootstrap.java:163)
2021-02-17T20:53:25.1406271Z [INFO] 	at io.quarkus.test.junit.QuarkusTestExtension.doJavaStart(QuarkusTestExtension.java:292)
2021-02-17T20:53:25.1410684Z [INFO] 	at io.quarkus.test.junit.QuarkusTestExtension.ensureStarted(QuarkusTestExtension.java:624)
2021-02-17T20:53:25.1413932Z [INFO] 	at io.quarkus.test.junit.QuarkusTestExtension.beforeAll(QuarkusTestExtension.java:670)
2021-02-17T20:53:25.1417739Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$8(ClassBasedTestDescriptor.java:368)
2021-02-17T20:53:25.1428466Z [INFO] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2021-02-17T20:53:25.1433279Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:368)
2021-02-17T20:53:25.1437864Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:192)
2021-02-17T20:53:25.1442122Z [INFO] 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
2021-02-17T20:53:25.1447320Z [INFO] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:136)
2021-02-17T20:53:25.1449785Z [INFO] 	... 31 more
2021-02-17T20:53:25.1452547Z [INFO] Caused by: java.lang.RuntimeException: Failed to create application model resolver for /home/runner/work/quarkus-freemarker/quarkus-freemarker/integration-tests/config-defaults
2021-02-17T20:53:25.1457137Z [INFO] 	at io.quarkus.bootstrap.BootstrapAppModelFactory.getAppModelResolver(BootstrapAppModelFactory.java:198)
2021-02-17T20:53:25.1461837Z [INFO] 	at io.quarkus.bootstrap.BootstrapAppModelFactory.resolveAppModel(BootstrapAppModelFactory.java:305)
2021-02-17T20:53:25.1464435Z [INFO] 	... 41 more
2021-02-17T20:53:25.1468986Z [INFO] Caused by: io.quarkus.bootstrap.resolver.maven.BootstrapMavenException: Failed to read artifact descriptor for io.quarkiverse.freemarker:quarkiverse-freemarker-integration-test-config-defaults:pom:0.2.0
2021-02-17T20:53:25.1476286Z [INFO] 	at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.resolveCurrentProjectRepos(BootstrapMavenContext.java:519)
2021-02-17T20:53:25.1481056Z [INFO] 	at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.resolveRemoteRepos(BootstrapMavenContext.java:479)
2021-02-17T20:53:25.1485625Z [INFO] 	at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.getRemoteRepositories(BootstrapMavenContext.java:231)
2021-02-17T20:53:25.1490972Z [INFO] 	at io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver.<init>(MavenArtifactResolver.java:128)
2021-02-17T20:53:25.1494721Z [INFO] 	at io.quarkus.bootstrap.BootstrapAppModelFactory.getAppModelResolver(BootstrapAppModelFactory.java:192)
2021-02-17T20:53:25.1497282Z [INFO] 	... 42 more
2021-02-17T20:53:25.1500947Z [INFO] Caused by: org.eclipse.aether.resolution.ArtifactDescriptorException: Failed to read artifact descriptor for io.quarkiverse.freemarker:quarkiverse-freemarker-integration-test-config-defaults:pom:0.2.0
2021-02-17T20:53:25.1506093Z [INFO] 	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:301)
2021-02-17T20:53:25.1511470Z [INFO] 	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.readArtifactDescriptor(DefaultArtifactDescriptorReader.java:171)
2021-02-17T20:53:25.1516812Z [INFO] 	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.readArtifactDescriptor(DefaultRepositorySystem.java:255)
2021-02-17T20:53:25.1521707Z [INFO] 	at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.resolveCurrentProjectRepos(BootstrapMavenContext.java:514)
2021-02-17T20:53:25.1524745Z [INFO] 	... 46 more
2021-02-17T20:53:25.1528652Z [INFO] Caused by: org.apache.maven.model.resolution.UnresolvableModelException: Could not find artifact io.quarkiverse.freemarker:quarkiverse-freemarker-integration-tests:pom:0.2.0 in central (https://repo.maven.apache.org/maven2)
2021-02-17T20:53:25.1533440Z [INFO] 	at org.apache.maven.repository.internal.DefaultModelResolver.resolveModel(DefaultModelResolver.java:176)
2021-02-17T20:53:25.1537540Z [INFO] 	at org.apache.maven.repository.internal.DefaultModelResolver.resolveModel(DefaultModelResolver.java:222)
2021-02-17T20:53:25.1541800Z [INFO] 	at org.apache.maven.model.building.DefaultModelBuilder.readParentExternally(DefaultModelBuilder.java:1070)
2021-02-17T20:53:25.1545718Z [INFO] 	at org.apache.maven.model.building.DefaultModelBuilder.readParent(DefaultModelBuilder.java:846)
2021-02-17T20:53:25.1549164Z [INFO] 	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:337)
2021-02-17T20:53:25.1552513Z [INFO] 	at io.quarkus.bootstrap.resolver.maven.MavenModelBuilder.build(MavenModelBuilder.java:53)
2021-02-17T20:53:25.1556633Z [INFO] 	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:292)
2021-02-17T20:53:25.1559538Z [INFO] 	... 49 more
2021-02-17T20:53:25.1563143Z [INFO] Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact io.quarkiverse.freemarker:quarkiverse-freemarker-integration-tests:pom:0.2.0 in central (https://repo.maven.apache.org/maven2)
2021-02-17T20:53:25.1567894Z [INFO] 	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:424)
2021-02-17T20:53:25.1571983Z [INFO] 	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:229)
2021-02-17T20:53:25.1576294Z [INFO] 	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact(DefaultArtifactResolver.java:207)
2021-02-17T20:53:25.1580618Z [INFO] 	at org.apache.maven.repository.internal.DefaultModelResolver.resolveModel(DefaultModelResolver.java:172)
2021-02-17T20:53:25.1583179Z [INFO] 	... 55 more
2021-02-17T20:53:25.1586653Z [INFO] Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact io.quarkiverse.freemarker:quarkiverse-freemarker-integration-tests:pom:0.2.0 in central (https://repo.maven.apache.org/maven2)
2021-02-17T20:53:25.1591492Z [INFO] 	at org.eclipse.aether.connector.basic.ArtifactTransportListener.transferFailed(ArtifactTransportListener.java:48)
2021-02-17T20:53:25.1595636Z [INFO] 	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:369)
2021-02-17T20:53:25.1599513Z [INFO] 	at org.eclipse.aether.util.concurrency.RunnableErrorForwarder$1.run(RunnableErrorForwarder.java:75)
2021-02-17T20:53:25.1603286Z [INFO] 	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$DirectExecutor.execute(BasicRepositoryConnector.java:644)
2021-02-17T20:53:25.1607280Z [INFO] 	at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get(BasicRepositoryConnector.java:262)
2021-02-17T20:53:25.1611352Z [INFO] 	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:499)
2021-02-17T20:53:25.1615438Z [INFO] 	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:401)
2021-02-17T20:53:25.1617869Z [INFO] 	... 58 more
2021-02-17T20:53:25.1618884Z [INFO] 
2021-02-17T20:53:25.4612174Z [INFO] [INFO] 
2021-02-17T20:53:25.4711035Z [INFO] [INFO] Results:
2021-02-17T20:53:25.4712197Z [INFO] [INFO] 
2021-02-17T20:53:25.4713279Z [INFO] [ERROR] Errors: 
2021-02-17T20:53:25.4716209Z [INFO] [ERROR]   ConfigDefaultsTest.dynamicTemplate » Runtime io.quarkus.bootstrap.BootstrapExc...
2021-02-17T20:53:25.4718215Z [INFO] [INFO] 
2021-02-17T20:53:25.4719396Z [INFO] [ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 1
2021-02-17T20:53:25.4720740Z [INFO] [INFO] 
2021-02-17T20:53:25.4722274Z [INFO] [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.4724136Z [INFO] [INFO] Reactor Summary for Quarkus - Freemarker - Parent 0.2.0:
2021-02-17T20:53:25.4725532Z [INFO] [INFO] 
2021-02-17T20:53:25.4727491Z [INFO] [INFO] Quarkus - Freemarker - Parent ...................... SUCCESS [01:36 min]
2021-02-17T20:53:25.4729415Z [INFO] [INFO] Quarkus - Freemarker - Runtime ..................... SUCCESS [ 31.844 s]
2021-02-17T20:53:25.4731307Z [INFO] [INFO] Quarkus - Freemarker - Deployment .................. SUCCESS [  6.733 s]
2021-02-17T20:53:25.4733259Z [INFO] [INFO] Quarkus - Freemarker - Integration tests ........... SUCCESS [  0.016 s]
2021-02-17T20:53:25.4735401Z [INFO] [INFO] Quarkus - Freemarker - Integration test - Configuration defaults FAILURE [  9.380 s]
2021-02-17T20:53:25.4737551Z [INFO] [INFO] Quarkus - Freemarker - Integration test - Legacy configuration SKIPPED
2021-02-17T20:53:25.4739582Z [INFO] [INFO] Quarkus - Freemarker - Integration test - Template sets SKIPPED
2021-02-17T20:53:25.4741527Z [INFO] [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.4743001Z [INFO] [INFO] BUILD FAILURE
2021-02-17T20:53:25.4744559Z [INFO] [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.4746050Z [INFO] [INFO] Total time:  02:25 min
2021-02-17T20:53:25.4747431Z [INFO] [INFO] Finished at: 2021-02-17T20:53:25Z
2021-02-17T20:53:25.4749067Z [INFO] [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.4752473Z [INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project quarkiverse-freemarker-integration-test-config-defaults: There are test failures.
2021-02-17T20:53:25.4755685Z [INFO] [ERROR] 
2021-02-17T20:53:25.4758132Z [INFO] [ERROR] Please refer to /home/runner/work/quarkus-freemarker/quarkus-freemarker/integration-tests/config-defaults/target/surefire-reports for the individual test results.
2021-02-17T20:53:25.4761288Z [INFO] [ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
2021-02-17T20:53:25.4763087Z [INFO] [ERROR] -> [Help 1]
2021-02-17T20:53:25.4764154Z [INFO] [ERROR] 
2021-02-17T20:53:25.4765753Z [INFO] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
2021-02-17T20:53:25.4768376Z [INFO] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
2021-02-17T20:53:25.4769808Z [INFO] [ERROR] 
2021-02-17T20:53:25.4771470Z [INFO] [ERROR] For more information about the errors and possible solutions, please read the following articles:
2021-02-17T20:53:25.4773705Z [INFO] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
2021-02-17T20:53:25.4775532Z [INFO] [ERROR] 
2021-02-17T20:53:25.4776977Z [INFO] [ERROR] After correcting the problems, you can resume the build with the command
2021-02-17T20:53:25.4779590Z [INFO] [ERROR]   mvn <args> -rf :quarkiverse-freemarker-integration-test-config-defaults
2021-02-17T20:53:25.5031679Z [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.5035256Z [INFO] Reactor Summary for Quarkus - Freemarker - Parent 0.2.0-SNAPSHOT:
2021-02-17T20:53:25.5036703Z [INFO] 
2021-02-17T20:53:25.5039855Z [INFO] Quarkus - Freemarker - Parent ...................... FAILURE [03:02 min]
2021-02-17T20:53:25.5041663Z [INFO] Quarkus - Freemarker - Runtime ..................... SKIPPED
2021-02-17T20:53:25.5043394Z [INFO] Quarkus - Freemarker - Deployment .................. SKIPPED
2021-02-17T20:53:25.5045148Z [INFO] Quarkus - Freemarker - Integration tests ........... SKIPPED
2021-02-17T20:53:25.5048399Z [INFO] Quarkus - Freemarker - Integration test - Configuration defaults SKIPPED
2021-02-17T20:53:25.5053405Z [INFO] Quarkus - Freemarker - Integration test - Legacy configuration SKIPPED
2021-02-17T20:53:25.5068784Z [INFO] Quarkus - Freemarker - Integration test - Template sets SKIPPED
2021-02-17T20:53:25.5070771Z [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.5072217Z [INFO] BUILD FAILURE
2021-02-17T20:53:25.5073692Z [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.5075134Z [INFO] Total time:  03:51 min
2021-02-17T20:53:25.5076407Z [INFO] Finished at: 2021-02-17T20:53:25Z
2021-02-17T20:53:25.5077943Z [INFO] ------------------------------------------------------------------------
2021-02-17T20:53:25.5080886Z [ERROR] Failed to execute goal org.apache.maven.plugins:maven-release-plugin:2.5.3:prepare (default-cli) on project quarkus-freemarker-parent: Maven execution failed, exit code: '1' -> [Help 1]
2021-02-17T20:53:25.5083108Z [ERROR] 
2021-02-17T20:53:25.5084620Z [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
2021-02-17T20:53:25.5109285Z [ERROR] Re-run Maven using the -X switch to enable full debug logging.
2021-02-17T20:53:25.5110802Z [ERROR] 
2021-02-17T20:53:25.5112390Z [ERROR] For more information about the errors and possible solutions, please read the following articles:
2021-02-17T20:53:25.5114600Z [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
2021-02-17T20:53:25.5232713Z ##[error]Process completed with exit code 1.
```

see https://github.com/quarkiverse/quarkus-freemarker/actions/runs/576107665

when I compare with the `jberet` extension, the `integration-tests` module is actually part of the [normal module list](https://github.com/quarkiverse/quarkus-jberet/blob/master/pom.xml#L38).

but may be the situation is different.

Note that I was able to release 0.1.0 10 days ago because at the time we had only one `integration-test` module. so during the [release](https://github.com/quarkiverse/quarkus-freemarker/runs/1794538638?check_suite_focus=true) it was to build the test module:
```
2021-01-29T21:15:12.4153033Z [INFO] [INFO] ------------------------------------------------------------------------
2021-01-29T21:15:12.4155387Z [INFO] [INFO] Reactor Summary for Quarkus - Freemarker - Parent 0.1.0:
2021-01-29T21:15:12.4157616Z [INFO] [INFO] 
2021-01-29T21:15:12.4183838Z [INFO] [INFO] Quarkus - Freemarker - Parent ...................... SUCCESS [01:32 min]
2021-01-29T21:15:12.4186267Z [INFO] [INFO] Quarkus - Freemarker - Runtime ..................... SUCCESS [ 28.652 s]
2021-01-29T21:15:12.4188577Z [INFO] [INFO] Quarkus - Freemarker - Deployment .................. SUCCESS [  7.461 s]
2021-01-29T21:15:12.4190870Z [INFO] [INFO] Quarkus - Freemarker - Integration test ............ SUCCESS [ 31.470 s]
2021-01-29T21:15:12.4193156Z [INFO] [INFO] ------------------------------------------------------------------------
2021-01-29T21:15:12.4194919Z [INFO] [INFO] BUILD SUCCESS
2021-01-29T21:15:12.4196939Z [INFO] [INFO] ------------------------------------------------------------------------
2021-01-29T21:15:12.4198717Z [INFO] [INFO] Total time:  02:41 min
2021-01-29T21:15:12.4262519Z [INFO] [INFO] Finished at: 2021-01-29T21:15:12Z
2021-01-29T21:15:12.4269371Z [INFO] [INFO] ------------------------------------------------------------------------
```

so I am not sure what is the proper way to do it.
should I remove the `it` profile and just include the `integration-tests` module in the list with the other modules?

/cc @ppalaga @aloubyansky @gastaldi

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -53,7 +53,7 @@ jobs:
           mvn -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}} -s maven-settings.xml
           git checkout ${{github.base_ref}}
           git rebase release
-          mvn -B release:perform -Darguments=-DperformRelease -Prelease -s maven-settings.xml
+          mvn -B release:perform -Darguments=-DperformRelease -DperformRelease -Prelease -s maven-settings.xml
 
       - name: Push changes to ${{github.base_ref}}
         uses: ad-m/github-push-action@v0.6.0

